packages/flutter_tools/test/integration.shard/test_data/gen_l10n_project.dart

```diff
@@ -66,6 +66,29 @@ class Home extends StatelessWidget {
     final List<String> results = [];
     return Row(
       children: <Widget>[
+        Builder(
+          builder: (BuildContext context) {
+            results.add('--- supportedLocales tests ---');
```

Comment:
Need to return a Container here. Given the number of times this idiom appears, it would be simpler to factor out a Widget with a void callback

Reply:
I factored out a widget that does this and prints the l10n END string if an exception is thrown. PTAL